### ISSUE TITLE
Gnome Theme Resets Every Time Settings->Appearance is Opened

### ISSUE THREAD

**Issue by graxl** [pop-os/pop#817]
**Distribution (run `cat /etc/os-release`):**
 
NAME="Pop!_OS"
VERSION="19.10"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.10"
VERSION_ID="19.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=eoan
UBUNTU_CODENAME=eoan
LOGO=distributor-logo-pop-os

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
gnome-tweaks, settings, gtk, gnome, application themes

**Issue/Bug Description:**
I've installed various UI customizations vimix gtk themes, paper icon theme, nerd fonts, oh my zsh, and a couple of google ttf fonts I like.
I've then gotten everything configured
As soon as I open Settings->Appearance the application theme resets from any non-pop theme (in this case vimix-dark-laptop to pop but I've tried it with others and the reset still happens).
If I left the settings app on the appearance tab then the reset happens as soon as it opens

**Steps to reproduce (if you know):**
See description

**Expected behavior:**
I would expect me to have to make some sort of change rather than have it auto reset.

**Other Notes:**
As best I can tell it leaves the icons, cursor, fonts, etc alone and only affects the applications theme.

**Comment by pawel-szopinski** [comment]
I have the same Pop Os version and experience the same issue. My friend's pop os installation is also affected by this bug.

**Comment by Alexandre-Fernandez** [comment]
same on Ubuntu 22.10

**Comment by MatthiasLohr** [comment]
> same on Ubuntu 22.10

Yep, can confirm that. @Alexandre-Fernandez, are you aware of any open issue tracking that with Ubuntu? No idea where to search for or create that...
